xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -76,14 +76,11 @@ async def send_reliable_submission(
         XRPLReliableSubmissionException: if the transaction fails or is missing a
```

Comment:
This comment need to be updated since this no longer throws if the transaction fails.

Reply:
(Or does _wait_for_final_transaction_outcome throw if the transaction fails now?)

Reply:
"fails" also refers to the last ledger sequence passing.